store/store.go

```diff
@@ -193,7 +197,20 @@ func (s *store) RecentBlockByStamp(stamp hash.Stamp) (uint32, *block.Block) {
 	return 0, nil
 }
 
-func (s *store) Transaction(id tx.ID) (*StoredTx, error) {
+func (s *store) PublicKey(addr crypto.Address) (crypto.PublicKey, bool) {
```

Comment:
Let's return an  error here. Also to prevent unnecessary casting return type can be `*bls.PublicKey`